changeppbot.py

```diff
@@ -53,20 +56,21 @@ def catch_basic_responses(msg, email, db):
         return help_response()
 
 
-def pre_process(msg, email, db):
+def pre_process(msg: str, email, db):
     is_admin = False
     if msg.startswith('admin '):
         is_admin = True
 
     return {
         "is_admin": is_admin,
-        "id": db.get_mavenlink_id(email)
     }
 
 
 def process(client: SocketModeClient, req: SocketModeRequest):
     """This function handles different events from the Slack API and is the starting point for everything that IncludeBot does"""
 
+    print("\n\n\nReceived request: " + str(req))
```

Comment:
this isn't how we should handle logging, see line 107 of this file

Reply:
fixed